scripts/sonobuoy-test.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 MODE=$1
-
+export PATH=$PATH:$HOME/go/bin
```

Comment:
this is safer and won't break if there's spaces added in the path somewhere
```suggestion
export PATH="${PATH}:${HOME}/go/bin"
```